src/server/helpers/source-api-request.ts

```diff
@@ -178,6 +194,25 @@ export async function requestData<T>(
 
     const request: AxiosPromise<T> = axiosRequest.request(requestConfig);
     const response: AxiosResponse<T> = await request;
+
+    // if we have a next link
+    if (
+      response.headers?.link?.indexOf('rel="next"') > 0 &&
+      typeof requestConfig.mergeResults === 'function'
```

Comment:
```suggestion
      typeof requestConfig.mergeResults === 'function' &&
      response.headers?.link?.includes('rel="next"')
```

De meest 'goedkope' check kun je beter eerst plaatsen. `indexOf()` werkt overigens met `> -1` om te checken op aanwezigheid. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf